PredictiveMethods/CaseOutcome/PredictiveMonitor.py

```diff
@@ -65,4 +65,4 @@ def test(self, dt_test):
         for label_col_idx, label in enumerate(pred_model.cls.classes_):
             if label == self.pos_label:
                 # output probability of a positive class
-                print(np.round(predictions_proba[0, label_col_idx], decimals=3))
+                return np.round(predictions_proba[0, label_col_idx], decimals=3)
```

Comment:
I agree that components should be decoupled, thus, we can build microservices architecture. However, note that predictions are sent to UI as soon as they all available. We cannot send them one by one as clients will see inconsistent data (will not be clear whether all results are up-to-date). From the other side, as soon as server receives event, it already can be used to update all relevant clients. Just be sure that those conditions are satisfied.